### DISCUSSION TITLE
Notebook tutorial examples

### DISCUSSION THREAD

**Discussion by rubiop** [MeasureTransport/MParT#145] [General]
I was wondering how we should proceed to synchronize on #80. I think the idea (at least for the JOSS paper) is to demonstrate code capabilities and how to use it. I don't think we want examples as application developed as in [https://github.com/MeasureTransport/SIAMannual22](url) (but not sure). 

Also I think the idea is to have exactly the same notebooks for all the different languages so I guess we want to list what examples we want to provide. For coordination, I think an idea to keep the same name of variables, explanations etc. between all the languages is to first all contribute to complete the notebooks for one language (e.g. python) and then, at the very end, convert them to other languages.

Let me know if you have other strategies in mind to do that! Feel free to also list what you think tutorials should cover.

(edit: this is related to #16)

**Comment by mparno** [comment]
One other thing we might want to coordinate on between the languages (at least julia and python) is how we want to import MParT.   I have  mixed feelings about using `from MParT import *` and typically prefer something like `import MParT as mp` to avoid conflicts with other packages.   What do you all think?   

Users will build off these examples by copy/pasting so whatever we choose will likely become the de facto standard for using MParT.   Numpy examples almost always use `import numpy as np` and the standard practice is therefore to use this form.  Fenics on the other hand, typically uses `from dolfin import *`, so this has become standard.
